{"name": "test_vwologin_pf", "status": "broken", "statusDetails": {"message": "RecursionError: maximum recursion depth exceeded in comparison", "trace": "self = <tests.vwoLoginTests.test_vwologin_pf.TestVWOLogin object at 0x1110c2210>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"67474a289fa9fe055453b5f3a0f8fc24\")>\n\n    @pytest.mark.positive\n    def test_vwologin_pf(self, driver):\n        loginPage = LoginPage(driver)\n>       loginPage.login_to_vwo(\"contact+augg@thetestingacademy.com\", \"Wingify@123\")\n\ntest_vwologin_pf.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../venv/lib/python3.11/site-packages/seleniumpagefactory/Pagefactory.py:50: in __getattr__\n    if loc in self.locators.keys():\n../../venv/lib/python3.11/site-packages/seleniumpagefactory/Pagefactory.py:50: in __getattr__\n    if loc in self.locators.keys():\n../../venv/lib/python3.11/site-packages/seleniumpagefactory/Pagefactory.py:50: in __getattr__\n    if loc in self.locators.keys():\nE   RecursionError: maximum recursion depth exceeded in comparison\n!!! Recursion detected (same locals & position)"}, "start": 1693220224309, "stop": 1693220224311, "uuid": "5db5977b-dfcc-41b0-87c6-a46b6b0bdb2c", "historyId": "d785033fdc4331a498903d28e4bca011", "testCaseId": "d785033fdc4331a498903d28e4bca011", "fullName": "tests.vwoLoginTests.test_vwologin_pf.TestVWOLogin#test_vwologin_pf", "labels": [{"name": "tag", "value": "positive"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwologin_pf"}, {"name": "subSuite", "value": "TestVWOLogin"}, {"name": "host", "value": "MBP-C02DRR20MD6M"}, {"name": "thread", "value": "94309-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwologin_pf"}]}